frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/volume-wrapper.ts

```diff
@@ -126,6 +126,9 @@ export class VolumeWrapper extends ObjectWithTypePropertyWrapper<
 
   getReferencedObject = () => getVolumeReferencedObject(this);
 
+  isEnvVolumeType = () =>
```

Comment:
this one redundant now... need to remove

Reply:
+1

Reply:
please remove (: